### PR TITLE
Adding a simple method to get the full fulfillment information for an…

### PR DESCRIPTION
**Pull request by Hariador**
… order from Shopify

### CODE REVIEW

File: src/Services/Fulfillment.php [hunk 1]
```diff
@@ -20,6 +20,14 @@ public function create(ShopifyFulfillment $fulfillment)
         return $this->unserializeModel($raw['fulfillment'], ShopifyFulfillment::class);
     }
 
+    //Needed some way to get all of the fulfillment information about an order from Shopify
```
Comment (Wryte) [inline src/Services/Fulfillment.php]: This should be a proper docblock comment.

File: src/Services/Fulfillment.php [hunk 2]
```diff
@@ -20,6 +20,14 @@ public function create(ShopifyFulfillment $fulfillment)
...
+    public function get($orderID)
+    {
+        $raw = $this->client->get("admin/orders/{$orderID}/fulfillments.json", []);
+
+        return $raw['fulfillments'];
```
Comment (bmansell) [inline src/Services/Fulfillment.php]: This should unserialize into a collection of fulfillment models here:

```php
$fulfillments = array_map(function ($fulfillment) {
    return $this->unserializeModel($fulfillment, ShopifyFulfillment::class);
}, $raw['fulfillments']);

return new Collection($fulfillments);
```

Comment (Hariador) [reply src/Services/Fulfillment.php]: You can't. The response isn't just a list of fulfillments, it's all the fulfillment information about the order, so there's a bunch of top level meta data, and a collection of actual fulfillment objects.  Also, I don't think our fulfillment object matches what shopify actually sends anymore

Comment (Hariador) [reply src/Services/Fulfillment.php]: Turns out I was wrong.  I blame something.

File: src/Services/Fulfillment.php [hunk 3]
```diff
@@ -20,6 +20,22 @@ public function create(ShopifyFulfillment $fulfillment)
         return $this->unserializeModel($raw['fulfillment'], ShopifyFulfillment::class);
     }
 
+    /**
+     * @param string $orderId
+     *
+     * @return ShopifyFulfillment | collection
```
Comment (bmansell) [inline src/Services/Fulfillment.php]: The `C` of collection should be uppercase.

**Pull request closed without merging**